src/list.scss

```diff
@@ -387,4 +427,158 @@ $block: #{$fd-namespace}-list;
       }
     }
   }
+
+  &--links {
+    .#{$block}__item {
+      padding: 0;
+    }
+  }
+
+  // Standard List with Byline
+  &--byline {
+    .#{$block}__item {
+      width: 100%;
+      height: $fd-list-byline-item-height;
+      padding: $fd-list-byline-item-padding-x $fd-list-byline-item-padding-y;
+      align-items: flex-start;
+    }
+
+    .#{$block}__form-item {
+      margin-left: -1rem;
```

Comment:
value can be moved to variable since it can be reused
**Edit** there are few places when values can be moved to variables

Reply:
I will go over the values that are used more than once and put them in variables.
Edit2 comment: this is not a List with Navigation Indicator. This is just an example that the icon (ANY icon) can be placed on the right side. I can use another icon in the documentation